Unix/http/httpclientauth.c

```diff
@@ -1436,23 +1461,22 @@ HttpClient_EncryptData(_In_ HttpClient_SR_SocketData * handler, _Out_ Page **pHe
         iov[1].type = GSS_IOV_BUFFER_TYPE_DATA;
         iov[1].buffer.length = original_content_len;
         iov[1].buffer.value  = poriginal_data;
+        iov[2].type = 9 /*GSS_IOV_BUFFER_TYPE_PADDING*/  | GSS_IOV_BUFFER_FLAG_ALLOCATE;
```

Comment:
Fine! I didn't see the 9, just the commented out value!
Bruce is going to define that value in the header so it will be all nice again :)